src/Eccube/Service/CartService.php

```diff
@@ -509,10 +509,9 @@ public function downProductQuantity($productClassId)
     {
         $quantity = $this->getProductQuantity($productClassId) - 1;
 
+        //fix click continue minus button.
```

Comment:
this comment is unnecessary.
because,,,
- it is clear by looking at the code.
- "continue minus button" is just one scenario. ("fix click continue minus button" is not function's responsibility)